psscriptanalyzer/Deploy-DemoApp.ps1

```
@@ -0,0 +1,46 @@
+    }
+}
+
+function Get-Users {
+  $users = gci C:\app\users -Filter *.json
+  foreach ($user in $users) {
+      $content = cat $user.FullName
+      Write-Host $content
+  }
+}
+
+function Update-Service {
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'Update-Service' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)